docs/docs/references/configuration/cli/trivy_aws.md

```diff
@@ -76,6 +76,8 @@ trivy aws [flags]
       --endpoint string                   AWS Endpoint override
       --exit-code int                     specify exit code when any security issues are found
   -f, --format string                     format (table,json,template,sarif,cyclonedx,spdx,spdx-json,github,cosign-vuln) (default "table")
+      --helm-api-versions strings         specify Helm Kubernetes api-version used for Capabilities.APIVersions (can specify multiple or separate values with commas: policy/v1/PodDisruptionBudget,apps/v1/Deployment)
+      --helm-kube-version string          specify Helm kubernetes version
```

Comment:
I think the description of the flags should specify the helm flags of the `template` command with which they are mapped.

Reply:
```suggestion
      --helm-kube-version string          specify Helm Kubernetes version for the helm template command
      --helm-api-versions strings         specify Helm Kubernetes api-version used for Capabilities.APIVersions for the helm template command (can specify multiple or separate values with commas: policy/v1/PodDisruptionBudget,apps/v1/Deployment)",
```

Like this? @nikpivkin

Reply:
How about the next one:

Kubernetes version used for `Capabilities.KubeVersion`. This flag is the same as the `kube-version` flag of the `helm template` command.

The same is true for `api-versions`. This will allow the user to understand the purpose of the flag if the user is already familiar with helm or can refer to the helm documentation.. Wdyt?